src/main/java/gdx/liftoff/ui/panels/CompleteButtonsPanel.java

```diff
@@ -126,6 +140,15 @@ private static String findManuallyInstalledIntellijOnWindows() throws Exception
         return new File(intellijFolder, "bin/idea64.exe").getAbsolutePath();
     }
 
+    @NotNull
+    private static List<String> getManuallyInstalledLinuxIdeaNames() {
```

Comment:
Why is this a function that makes a new List, rather than a static constant? It only gets called once either way in most cases, so it doesn't make much difference... It just strikes me as odd.

Reply:
For better readability, I moved this list to the static field level